test/db/abi/compilers/pelles_c

```diff
@@ -26,7 +26,7 @@ EXPECT=<<EOF
    38 * relocs
     4 * sections
     0 * segments
-  117 * strings
+  173 * strings
```

Comment:
Running rz-bin -zz here it seems that the new PR finds less strings.

Reply:
where?
![image](https://user-images.githubusercontent.com/561184/160207933-8f2d4403-4bfc-404f-a6b4-609753a435ac.png)